controller/volume_rebuilding_controller.go

```diff
@@ -73,6 +74,17 @@ func NewVolumeRebuildingController(
 	}
 	vbc.cacheSyncs = append(vbc.cacheSyncs, ds.VolumeInformer.HasSynced)
 
+	if _, err = ds.SettingInformer.AddEventHandlerWithResyncPeriod(cache.FilteringResourceEventHandler{
+		FilterFunc: isSettingRelatedToOffLineRebuilding,
+		Handler: cache.ResourceEventHandlerFuncs{
+			AddFunc:    vbc.enqueueSetting,
+			UpdateFunc: vbc.enqueueSettingUpdated,
```

Comment:
Why not use `func(old, cur interface{}) { sc.enqueueSetting(cur) }`?

Reply:
Modified to use `func(old, cur interface{}) { sc.enqueueSetting(cur) }`
Thanks.